satpy/tests/test_scene.py

```diff
@@ -826,7 +826,7 @@ def test_all_datasets_one_reader(self):
         num_reader_ds = 21 + 6
         assert len(id_list) == num_reader_ds
         id_list = scene.all_dataset_ids(composites=True)
-        assert len(id_list) == num_reader_ds + 29
+        assert len(id_list) == num_reader_ds + 33
```

Comment:
Is this expected? Other than just making the test pass, why did you other changes add more datasets here? Because they are in `visir.yaml`?

Reply:
I don't know.  I wanted to ask you.  Didn't you write this code and test?

The PR adds four composites to `visir.yaml`, should we expect this to increase the number to be tested against here?